websites/docs-smoke-test/src/content/index.mdx

```diff
@@ -29,3 +31,27 @@ title: Explore how to build a documentation website for commercetools
   <Card>Avengers: Age of Ultron</Card>
   <Card>Avengers: Infinity War</Card>
 </Cards>
+
+# Course cards
+
+<Cards narrow clickable>
+  <CourseCard duration="30 min" title="Course name" href="self-learning/overview" courseId="66">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+  <CourseCard duration="30 min" title="Course name" href="/">Course description. Introduction to extensibility possibilities available in Composable Commerce. </CourseCard>
+</Cards>
```

Comment:
This is really nice for authors :-) 

For smoke testing purposes you may want to include a card where the author chose to put extra blank lines and  ( or markdown into the card to see the layout effects.